src/main/java/org/mule/extension/http/api/request/builder/HttpRequesterRequestBuilder.java

```diff
@@ -160,7 +160,7 @@ public HttpRequestBuilder toHttpRequestBuilder(HttpRequesterConfig config) {
   }
 
   @Override
-  public TypedValue<?> getBodyAsTypedValue() {
+  public TypedValue getBodyAsTypedValue() {
```

Comment:
deletting the generic doesn't address the original comment. The real question is why do we want something that returns object instead of using a well defined type. Also, by looking at the implementation, it looks like you return a TypedValue for which the mediaType is always ANY. That's also a design smell.